satellite-software/src/BME680Sensor.cpp

```diff
@@ -650,15 +654,20 @@ uint8_t BME680Sensor::handleCommand(uint16_t cmdSequenceID, uint8_t command, uin
         if ((plate_temp < 200)  || (plate_temp > 400)) {
             return CMD_STATUS_BAD_PARAM;
         }
-        PRINT_DEBUG(F("  Gas heat temp = "));
+        PRINT_DEBUG(F("  Heat temp = "));
         PRINT_DEBUG(plate_temp);
         PRINT_DEBUG(F("\n"));
         setGasHeatTemperature(plate_temp);
+    }else if (command == 0x07){
+        PRINT_DEBUG(F("  default "));
+        PRINT_DEBUG(F("\n"));
```

Comment:
You might as well turn these two lines into a single `PRINTLN_DEBUG`.

Reply:
Done